### DISCUSSION TITLE
Question: is there a way to remove tag fragments from the URL?

### DISCUSSION THREAD

**Discussion by jrub** [rohit-gohri/redocusaurus#283] [Q&A]
We used tags and Redoc's `x-traitTag` to create groups of endpoints on the sidebar.

Now, all URLs have this `#tag` fragment, like in https://developers.quaderno.io/api/#tag/Invoices/operation/createInvoice

I'd love to remove that in order to simplify the URLs. Anyone knows how, or even if that's possible?

**Comment by rohit-gohri** [comment]
I was going to point to https://github.com/Redocly/redoc/issues/447 but then saw that you already commented there 😅

**Reply by jrub** [reply]
Thanks @rohit-gohri . Anyway if I understand correctly, the proposed code there doesn't remove the "#tag".
